packages/tasks/src/model-libraries.ts

```diff
@@ -358,6 +365,13 @@ export const MODEL_LIBRARIES_UI_ELEMENTS = {
 		filter: false,
 		countDownloads: `path:"tokenizer.model"`,
 	},
+	"sae": {
+		prettyLabel: "SAE",
+		repoName: "sae",
+		repoUrl: "https://github.com/EleutherAI/sae",
+		filter: true,
+		countDownloads: `path_extension:"safetensors" OR path_extension:"bin"`,
```

Comment:
Maybe let's use the `embed_tokens/cfg.json` file? https://huggingface.co/EleutherAI/sae-llama-3-8b-32x/tree/main/embed_tokens

Reply:
Like the other example, this was me guessing. To make sure I understand best practices, we're looking for a file path pattern that
1. Appears in every model
2. Only matches one file in each model

Is that correct? Perhaps the easiest solution is to just create a new metadata file which (for now, at least) primarily exists to support the integration.

Reply:
To be more precise, we are looking for a file path pattern that:
1. Appears in every model
2. Only matches one file when loading the model locally.

For example, on [Meta-Llama-3.1-8B-Instruct-GGUF](https://huggingface.co/lmstudio-community/Meta-Llama-3.1-8B-Instruct-GGUF/tree/main) any download of a `*.gguf` file is counted because we know that users won't download all GGUF files when instanciating the model (i.e. 1 usage == 1 file download).